src/WallGo/__init__.py

```diff
@@ -34,14 +34,13 @@
 try:
     import WallGoCollision
 
-    print(f"Loaded WallGoCollision package from location: {WallGoCollision.__path__}")
+    #print(f"Loaded WallGoCollision package from location: {WallGoCollision.__path__}")
     _bCollisionModuleAvailable = True  # pylint: disable=invalid-name
 
     from .collisionHelpers import *
 
 except ImportError as e:
-    print(f"Error loading WallGoCollision module: {e}")
-    print(
+    RuntimeWarning(f"Error loading WallGoCollision module: {e}"
```

Comment:
Should this not be `raise RuntimWarning(...`?

Reply:
I think it has to be `warnings.warn(...)`, otherwise `raise` would stop the program.